src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -250,6 +250,9 @@ void pegasus_mutation_duplicator::duplicate(mutation_tuple_set muts, callback cb
             // Because DUPLICATE comes from other clusters should not be forwarded to any other
             // destinations. A DUPLICATE is meant to be targeting only one cluster.
             continue;
+        } else if (rpc_code == dsn::apps::RPC_RRDB_RRDB_BULK_LOAD) {
```

Comment:
How about defining a set which contains RPC_RRDB_RRDB_DUPLICATE and RPC_RRDB_RRDB_BULK_LOAD? Please add some more comments as well.